src/wollok/lang.wlk

```diff
@@ -1069,6 +1082,9 @@ class Set inherits Collection {
    *     set.add(2)   => set = #{2, 3}, second add produces no effect   
    */
   override method add(element) native
+
+  /** @private */
+  method unsafeAdd(element) native
```

Comment:
How should we implement this behavior?

Reply:
To make a long story short: I need the add that checks if the element is present before adding and the actual addition to be separate methods so I don't have to add yet another bytecode instruction just for this scenario. So we could either make the add reference the unsafeAdd and let the unsafeAdd do the adding, or you could just implement an empty native, since it's not being used from anywhere else and I marked it as private it should not be that much of an issue. Ideally nobody ever sees or uses this method.